src/Numerics/SystemSolvers/columnwise_lu_solver.jl

```diff
@@ -52,9 +52,26 @@ elem_band(
 single_column(
     ::DGColumnBandedMatrix{D, P, NS, EH, EV, EB, SC},
 ) where {D, P, NS, EH, EV, EB, SC} = SC
+
+# DG: lower_bandwidth is Nq_v*nstate * eband - 1, (does not include itself) 
+# eband = 1 for inviscid, since nodal point at the face communicates to the overlaping point to its neighbor
+# and other points only communicates to points in the same element
+# eband = 2 for visous
+#
+# FV: lower_bandwidth is nstate * (stencil_width + 1 + 1) - 1,  
+# since the reconstruction states depend on stencil_width points on each side, 
+# and the flux depends on stencil_width + 1 points on each side
+# eband = (stencil_width + 2) for inviscid
+# eband = max{ (stencil_width + 2), 3} for viscid, 
+# since the viscous flux is computed by applying first order FD twice
+#  
+# The lower_bandwidth ls formulated as Nq_v*nstate * eband - 1
+lower_bandwidth(N, nstate, eband) = (N + 1) * nstate * eband - 1
```

Comment:
```suggestion
# The lower_bandwidth ls formulated as Nq_v * nstate * eband - 1
Nq_v = N + 1
lower_bandwidth(N, nstate, eband) = Nq_v * nstate * eband - 1
```
I think this might just read a bit nicer with the comment above and might be a bit an easier refactoring if/when we implement overintegration where `Nq_v = N + 1 + qextra`, where `qextra` witll be the number of extra quadrature points